.github/workflows/get-environment.yml

```
@@ -57,8 +61,20 @@ jobs:
       os_and_database_matrix: ${{ steps.get_os_database_matrix.outputs.result }}
       is_targeting_feature_branch: ${{ steps.get_stability.outputs.is_targeting_feature_branch }}
       is_nightly: ${{ steps.get_nightly_status.outputs.is_nightly }}
+      pull_request_environment: ${{ steps.pull_request_environment.outputs.pull_request_environment }}
 
     steps:
+      - name: Get pull request environment
+        id: pull_request_environment
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:62: Double quote to prevent globbing and word splitting

(shellcheck)